src/dotnet-test-xunit/Program.cs

```diff
@@ -181,34 +182,32 @@ static void OnUnhandledException(object sender, UnhandledExceptionEventArgs e)
         static List<IRunnerReporter> GetAvailableRunnerReporters()
         {
             var result = new List<IRunnerReporter>();
-            var runnerPath = Path.GetDirectoryName(AppContext.BaseDirectory);
+            var dependencyModel = DependencyContext.Load(typeof(Program).GetTypeInfo().Assembly);
```

Comment:
This should actually be the the test project so that any reporters specified as dependencies in the test project light up here. Unfortunately the argument parsing is intractably linked to this piece of code running prior to it and I didn't have the time to refactor it.